LandLord-core/src/main/java/biz/princeps/landlord/commands/management/Borders.java

```diff
@@ -69,7 +69,7 @@ public void onCommand(Properties properties, Arguments arguments) {
             int refreshRate = plugin.getConfig().getInt("Borders.refreshRate");
             this.tasks.put(p.getUniqueId(), new BukkitRunnable() {
                 int counter = 0;
-                int timeout = plugin.getConfig().getInt("Borders.timeout");
+                final int timeout = plugin.getConfig().getInt("Borders.timeout");
```

Comment:
Local final. Can't be accessed anyway.
Maybe excluding this to an own class would be cleaner. also since the run method contains a hell of nesting. But thats probably just code stile

Reply:
If extracting, I'd prefer implementing `Runnable` over extending `BukkitRunnable`

Reply:
Sure

Reply:
Right @SirYwell, however ``BukkitRunnable`` is necessary to cancel the task with the current implementation.